edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -402,15 +402,15 @@ label[for=register-password]{
   }
 
   .signin-logo {
-    width: 300px;
+    width: 270px;
   }
 
   .background-login {
     display: none;
   }
 
   .signin-container {
-    top: 260px;
+    top: 230px;
```

Comment:
Is there a way to prevent all these positioning values (top, left, right) from being static values? @luismorenolopera

Reply:
@Squirrel18 I think we can do this for the login-container class